Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM circleci/openjdk:8-jdk
+
+
+USER root
+
+RUN wget https://dl.google.com/dl/cloudsdk/channels/rapid/google-cloud-sdk.zip && unzip google-cloud-sdk.zip && rm google-cloud-sdk.zip
+RUN google-cloud-sdk/install.sh --usage-reporting=true --path-update=true --bash-completion=true --rc-path=/.bashrc --additional-components app cloud-datastore-emulator
+ENV PATH /google-cloud-sdk/bin:$PATH
+
+RUN gcloud components update --quiet && gcloud components install beta --quiet
```

Comment:
Do we need `beta` for anything? If not, we should remove that to avoid accidentally relying on beta software.

Reply:
I think that was me copy pasting stuff :P